docs/administration/fabrics/prophecy-fabrics/connections/sharepoint.md

```diff
@@ -16,3 +26,32 @@ To create a connection with Sharepoint, enter the following parameters:
 | Client ID                                                                 | Your SharePoint Client ID                                                                       |
 | Client Secret ([Secret required](docs/administration/secrets/secrets.md)) | Your SharePoint Client Secret                                                                   |
 | Site URL                                                                  | URL of the SharePoint site to connect (e.g., `https://yourcompany.sharepoint.com/sites/mysite`) |
+
+## Sharepoint permissions
+
+When you create a Sharepoint connection in Prophecy, access permissions are tied to the credentials you use. This means you will only see the files and folders your Sharepoint credentials have permission to access. Any actions you perform—such as reading or writing files—are done using those credentials.
```

Comment:
The permission setup is kinda tricky for sharepoint. User needs to follow this doc to create Client ID and Client Secret: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app

And then they need to make sure that their Microsoft account's admin grants this permission to the Application that they just created: "Sites.Manage.All"